content/_releases/23/31.md

```diff
@@ -0,0 +1,124 @@
+---
+title: Product Update - v23.31
+description: Details of the product update v23.31 from 1st of August 2023.
+layout: rel
+version: "23.31"
+section: Product Updates
+category: product-updates
+releaseDate: 2023-08-01
+redirect_from:
+  - /releases/23.31.html
+---
+
+## New Features
+### Dashboard Redesign:
+* Added Ordinary Flows counter - *click on the counter to open `Flows` page with filtered ordinary flows*
+* Added Real-time Flows counter - *click on the counter to open `Flows` page with filtered real-time flows*
+* Redesigned the current Execution Statistics graph - *each day on the graph is clickable and leads users to the corresponding page*
+* Added new graph for Container errors - *each day on the graph is clickable and leads users to the corresponding page*
+* Transferred Workspace Memory Usage graph
+
+### Transfer modal windows to the new styles:
+* Transform UI for Flow Designer: Configuration for Content-Based Router Component to React - Now users can map values from previous steps for evaluation.
+
+{% include img.html max-width="100%" url="/assets/img/RN/2331/BCR.png" title="BCR" %}
+
+{% if site.data.tenant.name == "elastic.io" %}
+
+## OEM Related
+### Removed `LOOKOUT_PROCESS_DATA` env var
+`LOOKOUT_PROCESS_DATA` was introduced to disable Runlog for the dashboard. After redesigning the Dashboard page, this env var does not do anything on the frontend, since runlog is not on the new dashboard.
+
+We've removed this env var from the frontend completely, and removed it from a lookout by making it run as if this env var is set to "false".
+
+### Change elasticsearch index config:
+
+We've improved the Logs search to better handle partial text matching (e.g. "test.123" log line previously would match only if a user searched for exact "test.123", but now both "test" and "123" will match the logline).
```

Comment:
The search function is now more user friendly as it now accepts partial matches (e.g.  ...